go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -43,10 +44,10 @@ type destination struct {
 	dnsName      string
 }
 
-func newEgressFirewall(egressFirewallPolicy *egressfirewallapi.EgressFirewall) *egressFirewall {
+func newEgressFirewall(originalEgressfirewall *egressfirewallapi.EgressFirewall) *egressFirewall {
```

Comment:
can this be `cloneEgressFirewall` or something like that?
I think readability would benefit. Also, the name clashes with `newEgressFirewall` - the variable used in some functions.

Another option is to unwrap the function in the only place it is used

Reply:
I'll unwrap it then, I didn't notice it was only called in one place. Thanks!

Reply:
Actually on second thought, unwrapping it will lead to conflicts between the argument name in the function where it is called and the struct name. I'll just call it `cloneEgressFirewall`